great_expectations/validator/validator.py

```diff
@@ -1425,7 +1417,7 @@ def get_expectation_suite(
              copy of _expectation_suite, not the original object.
         """
 
-        expectation_suite = copy.deepcopy(self._expectation_suite)
+        expectation_suite = copy.deepcopy(self.expectation_suite)
```

Comment:
When working with a class internals, should we be using getters/setters? I think the private variable is totally fine here and is actually the better practice.